src/wallet/rpcdump.cpp

```diff
@@ -616,9 +616,19 @@ UniValue dumpwallet(const JSONRPCRequest& request)
 
     EnsureWalletIsUnlocked(pwallet);
 
-    std::ofstream file;
     boost::filesystem::path filepath = request.params[0].get_str();
     filepath = boost::filesystem::absolute(filepath);
+
+    /* Prevent arbitrary files from being overwritten. There have been reports
```

Comment:
nit: I don't think this needs such a long comment. Just say we don't allow files to be overwritten for safety.

Reply:
Meh, better too long than too short.